modules/dnn/test/test_backends.cpp

```diff
@@ -162,6 +162,18 @@ TEST_P(DNNTestNetwork, MobileNet_SSD_Caffe)
                inp, "detection_out", "", diffScores);
 }
 
+TEST_P(DNNTestNetwork, MobileNet_SSD_Caffe_Different_Width_Height)
```

Comment:
This test is failed for me locally with `DNN_TARGET_MYRIAD`. May I ask you to take a look? The rest of test are OK.